docs/components/connectors/protocol/soap.md

```diff
@@ -0,0 +1,260 @@
+---
+id: soap
+title: SOAP Connector
+sidebar_label: SOAP Connector
+description: The SOAP Connector allows you to connect your BPMN process with SOAP services.
+---
+
+:::note
+The **SOAP Connector** is only supported by Self-Managed and Hybrid Camunda 8 instances.
```

Comment:
What do we mean when we say "Hybrid Camunda 8"? Is this the same as Self-Managed?

Reply:
Hybrid means that you are running the cluster in Saas, but your connector runtime is local.

Reply:
@akeller Do we have this defined anywhere, and I'm just missing it? If not, can add to glossary if this is a term we may use more often.

Reply:
@christinaausley we added it here previously: https://docs.camunda.io/docs/guides/use-connectors-in-hybrid-mode/

Reply:
I mean, introduced the term

Reply:
Ah! Got it. I may open an issue in the backlog to link to this from within the Connector docs and define this in our glossary. My mind went to badges, and wondering how we would flag things moving forward -- in this case, we can just lowercase hybrid and link to the hybrid mode guide when referenced in the Connector docs. WDYT?

Reply:
Works for me

Reply:
To my knowledge, Connectors is the only feature used in hybrid mode, but it would be great to have it in the glossary @christinaausley!